shell/common/rasterizer_unittests.cc

```diff
@@ -66,6 +66,7 @@ class MockExternalViewEmbedder : public ExternalViewEmbedder {
                PostPrerollResult(
                    fml::RefPtr<fml::RasterThreadMerger> raster_thread_merger));
   MOCK_METHOD0(GetCurrentCanvases, std::vector<SkCanvas*>());
+  MOCK_METHOD0(GetCurrentBuilders, std::vector<DisplayListBuilder*>());
```

Comment:
Test failed in here:
`note: unimplemented pure virtual method 'GetCurrentBuilders' in 'MockExternalViewEmbedder'`